Sources/LatiFlex/Views/Network/NetworkView/LatiFlexNetworkPresenter.swift

```diff
@@ -113,6 +116,7 @@ extension LatiFlexNetworkPresenter: LatiFlexNetworkPresenterInterface {
     }
     
     func textDidChange(searchtext: String) {
+        isSearching = true
         guard !searchtext.isEmpty else {
```

Comment:
you can move the clearbutton func logics to here. else case means searchtext is empty